assets/images/favicon.svg

```diff
@@ -3,7 +3,6 @@
     <!-- Generator: Sketch 63.1 (92452) - https://sketch.com -->
     <title>journal-symbol-no-hoop</title>
     <desc>Created with Sketch.</desc>
-    <g id="journal-symbol-no-hoop" stroke="none" stroke-width="1" fill="none" fill-rule="evenodd">
         <rect id="Rectangle" fill="#FFFFFF" x="0" y="0" width="500" height="500"></rect>
```

Comment:
https://github.com/elifesciences/journal/pull/1249/commits/acecfaa918d7ea73f76328153671b069bc9f04c7 needs undoing too doesn't it?